spec/requests/internal/users_spec.rb

```diff
@@ -66,35 +64,50 @@ def full_profile
   context "when merging users" do
     before do
       full_profile
-      post "/internal/users/#{user.id}/merge", params: { user: { merge_user_id: user2.id } }
-      Delayed::Worker.new(quiet: true).work_off
```

Comment:
`Delayed::Worker.new(quiet: true).work_off` it's not needed because the merge action doesn't really enqueue any jobs